src/test/browser/index.js

```diff
@@ -1,6 +0,0 @@
-// Entry point for tests that will be run in Electron (browser).
```

Comment:
Copied original comment below:

-// Entry point for tests that will be run in Electron (browser).
 -// Only use browser testing for tests that require a browser environment such
 -// as needing a full mount as opposed to shallow rendering.
 -// Add more tests by importing them here.